### PR TITLE
Add Gvalchca GA15.0

### PR DESCRIPTION
**Pull request by Gvalch-ca**
## Description

Add Gvalch'ca GA15.0 PCB to QMK. This PCB is an Alps standard 60% with flex cutouts and some weird layout support. I have ran 2 groupbuys of GA15.0 on keebmeup.com discord server, and now running the 3rd, preparing of open GB. 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new keyboard - GA15.0 by Gvalch'ca

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gvalchca/ga150/rules.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes    # Unicode
+LTO_ENABLE = yes
+
+LAYOUTS = all
```
Comment (drashna) [inline keyboards/gvalchca/ga150/rules.mk:26]: There is no "all" layout.

```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/gvalchca/ga150/rules.mk (hunk 1 above)
Comment (Gvalch-ca) [reply keyboards/gvalchca/ga150/rules.mk]: Deleted that.

File: keyboards/gvalchca/ga150/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Gvalchca GA15.0
+
+Alps 60% PCB with some weird layouts support. To update the firmware connect the PCB to your computer and press Reset button.
+![GA15.0 image](https://i.imgur.com/Z7tNlGD.jpeg)
+
+* Keyboard Maintainer: [Gvalchca](https://github.com/Gvalch-ca)  
+* Hardware Supported: Gvalchca GA15.0
+* Hardware Availability: [KeebMeUp](https://keebmeup.com)
+
+Make example for this keyboard (after setting up your build environment):
```
[the remark this answers is not in the document]
Comment (Gvalch-ca) [reply keyboards/gvalchca/ga150/readme.md:10]: Line 3 says 'press Reset button'. Hardware reset button.

**Review by drashna — APPROVED** [review]

File: keyboards/gvalchca/ga150/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSLS,          \
+  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,           \
+  KC_LSFT, KC_NUBS, KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_RSFT,          \
+  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_RCTRL, KC_RCTRL),
+
+[_FUNC] = LAYOUT(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
```
Comment (fauxpark) [inline keyboards/gvalchca/ga150/keymaps/default/keymap.c:37]:
```suggestion
  KC_ESC,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSLS, KC_BSPC,
  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSLS,
  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,
  KC_LSFT, KC_NUBS, KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_RSFT,
  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_RCTRL, KC_RCTRL),

[_FUNC] = LAYOUT(
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```

File: keyboards/gvalchca/ga150/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Gvalchca GA15.0
+
+Alps 60% PCB with some weird layouts support. To update the firmware connect the PCB to your computer and enter bootloader by pressing hardware Reset button near the MCU.
+![GA15.0 image](https://i.imgur.com/Z7tNlGD.jpeg)
```
Comment (fauxpark) [inline keyboards/gvalchca/ga150/readme.md:4]:
```suggestion

![GA15.0 image](https://i.imgur.com/Z7tNlGDh.jpeg)
```

File: keyboards/gvalchca/ga150/readme.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+* Keyboard Maintainer: [Gvalchca](https://github.com/Gvalch-ca)
```
Comment (fauxpark) [inline keyboards/gvalchca/ga150/readme.md:6]:
```suggestion
* Keyboard Maintainer: [Gvalchca](https://github.com/Gvalch-ca)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by mfkiiyd** [comment]
sorry, copy err

**Pull request merged**